src/box/sql/build.c

```diff
@@ -558,7 +552,7 @@ field_def_create_for_pk(struct Parse *parser, struct field_def *field,
 void
 sqlAddPrimaryKey(struct Parse *pParse)
 {
-	struct space *space = pParse->create_column_def.space;
+	struct space *space = pParse->space;
 	assert(space != NULL || pParse->src_list != NULL);
 	if (space == NULL)
```

Comment:
Addition to `parse_type` - I see there is an existing `enum ast_type`. I think this one should be renamed to `sql_stmt_type/sql_ast_type` and used instead of `parse_type`.